alacritty/src/macos/locale.rs

```diff
@@ -61,38 +58,15 @@ fn system_locale() -> String {
         // https://developer.apple.com/documentation/foundation/nslocale/1416263-localeidentifier?language=objc
         // https://developer.apple.com/documentation/foundation/nslocale/1643060-countrycode?language=objc
         // https://developer.apple.com/documentation/foundation/nslocale/1643026-languagecode?language=objc
-        let is_language_code_supported: bool =
-            msg_send![locale, respondsToSelector: sel!(languageCode)];
-        let is_country_code_supported: bool =
-            msg_send![locale, respondsToSelector: sel!(countryCode)];
-        let locale_id = if is_language_code_supported && is_country_code_supported {
-            let language_code: *const Object = msg_send![locale, languageCode];
-            let language_code_str = nsstring_as_str(language_code).to_owned();
-            let _: () = msg_send![language_code, release];
-
-            let country_code: *const Object = msg_send![locale, countryCode];
-            let country_code_str = nsstring_as_str(country_code).to_owned();
-            let _: () = msg_send![country_code, release];
-
-            format!("{}_{}.UTF-8", &language_code_str, &country_code_str)
+        let is_language_code_supported: bool = locale.respondsToSelector(sel!(languageCode));
+        let is_country_code_supported: bool = locale.respondsToSelector(sel!(countryCode));
+        if is_language_code_supported && is_country_code_supported {
+            let language_code = locale.languageCode();
+            #[allow(deprecated)]
```

Comment:
What's deprecated here? The `countryCode` function? Is there an alternative?

Reply:
[The docs](https://developer.apple.com/documentation/foundation/nslocale/1643060-countrycode?language=objc) literally just say it's deprecated, I don't know why.